verilog/CST/functions_test.cc

```diff
@@ -535,6 +535,34 @@ TEST(GetFunctionBlockStatement, GetFunctionBody) {
   }
 }
 
+TEST(FunctionCallTest, GetFunctionCallName) {
+  constexpr int kTag = 1;  // value doesn't matter
+  const SyntaxTreeSearchTestCase kTestCases[] = {
+      {""},
+      {"module m;\nendmodule"},
+      {"module m;\ninitial begin\n", {kTag, "f1"}, "();\nend\nendmodule"},
+      {"module m;\ninitial begin\n", {kTag, "pkg::f1"}, "();\nend\nendmodule"},
```

Comment:
How can I write the test for the first one? the returned leaves are separated.
No the second case is different and covered by another function because it's `kMethodCallExtension`

Reply:
Never mind the first question :)